.storybook/stories/playground/story-editor/header/buttons/preview.js

```diff
@@ -41,7 +46,9 @@ function PreviewButton() {
   const openPreviewLink = () => {
     saveStory();
 
-    const previewLink = `${window.location.origin}/iframe.html?id=playground-preview--default`;
+    const previewLink = getStorybookUrl(
+      'iframe.html?id=playground-preview--default'
+    );
```

Comment:
Something `@storybook/addon-links` could be used for?

Reply:
Okay, trying...

Reply:
It should be fine here for the preview, but the only problem is that we need to deal with the promise for updating `newStoryURL` in `config` using `hrefTo`, which is currently static and easy to pass. 

https://github.com/google/web-stories-wp/blob/c51869d083ff38a5f09240d18a7eb8898c6d742f/.storybook/stories/playground/dashboard/index.js#L126-L128

Reply:
For preview, it is easy though 

```js
const previewLink = await hrefTo('Playground/preview', 'default');
```

Reply:
But yeah `@storybook/addon-links` looks like a better choice, will think again.

Reply:
It certainly feels less hardcoded and more future proof should storybook ever change the URL structure.

Reply:
If we follow thier API we would need to call `linkTo` method using `onClick` for which we will have to make changes to the core dashboard.

So to keep things simple, I used the `hrefTo` from `@storybook/addon-links` for preview but  created a better utility function called  `linkHrefTo` using `toId` from `'@storybook/csf'` which is internally being used by `@storybook/addon-links`  to convert string to id.

Updated in https://github.com/google/web-stories-wp/pull/9644/commits/a79794fe060b7a3806d08bbd5338128be5f11d78#